example/resources/schema/Makefile

```diff
@@ -0,0 +1,18 @@
+# avro
+encode-json-2-avro:
+	avro-tools fromjson --codec snappy --schema-file ./avro/twitter_scheme.avsc ./avro/tweet1.json > ../data/avro-tweet/tweet1.snappy.avro
+	avro-tools fromjson --codec snappy --schema-file ./avro/twitter_scheme.avsc ./avro/tweet2.json > ../data/avro-tweet/tweet2.snappy.avro
+
+# protofub
+encode-text-2-proto:
+	mkdir cpp | true
+	protoc --cpp_out=cpp ./proto/info.proto | true
+	protoc --encode=Info ./proto/info.proto < ./proto/info1.text  > ../data/proto-message/info1.data
+	protoc --encode=Info ./proto/info.proto < ./proto/info2.text  > ../data/proto-message/info2.data
```

Comment:
Maybe add a "Required software" section? This is not standard on my machine at least 
![Uploading image.png…]()

Reply:
All the encoded data is already in the directory `data/` and in running example there is no need to do anything.

These Makefile commands are just an example of how to encode data using cli tools. 
I added a description in readme and error with a link where to download cli tools. Installation of these CLI tools is not necessary until you want to experiment with serialization/deserialization (encoding/decoding).

Reply:
Got it. LGTM